Makefile

```diff
@@ -160,9 +160,14 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 	rm Dockerfile.cross
 
 ##@ Deployment
+## settings parameter for deployment 
+
+# getting the cloud id and the IngressHost 
+
+CLOUD_ID := $(shell oc get clusterversion -o jsonpath='{.items[].spec.clusterID}{"\n"}')
```

Comment:
These new lines can all be removed.  See comments in the metadata-server.yaml file.